cpp/src/parquet/column_writer.cc

```diff
@@ -1482,15 +1538,14 @@ class TypedColumnWriterImpl : public ColumnWriterImpl, public TypedColumnWriter<
 
   void WriteValuesSpaced(const T* values, int64_t num_values, int64_t num_spaced_values,
                          const uint8_t* valid_bits, int64_t valid_bits_offset,
-                         int64_t num_levels) {
+                         int64_t num_levels, int64_t num_nulls) {
```

Comment:
we should clarify what num_nulls means here.  We've had problem in the past on interpretation of empty-lists as nulls (I believe this is what the parquet spec expects), and num_null I think it usually derived from Arrow's concept but i could be mistaken.

Reply:
You're right. I have added a doc string to explain them.